tensorflow/python/keras/benchmarks/BUILD

```diff
@@ -109,3 +109,95 @@ py_test(
         "//tensorflow:tensorflow_py",
     ],
 )
+
+py_test(
+    name = "antirectifier_benchmark_test",
+    size = "medium",
+    srcs = ["keras_examples_benchmarks/antirectifier_benchmark_test.py"],
+    python_version = "PY3",
+    tags = [
+        "no_pip",  # b/161253163
+        "no_windows",  # b/160628318
```

Comment:
Can you make those two tags as a constant tag and used by each py_test? Then there won't be too many bug numbers in the build file?

Reply:
I didn't see any kind of usage in bazel. Also, I checked the tensorflow repo and they used the duplicate tags. You can check [here](https://github.com/tensorflow/tensorflow/blob/822b035fd1382128eb8ac65b9a89c2348b8fb7cf/tensorflow/compiler/tests/BUILD#L260), also [here](https://github.com/tensorflow/tensorflow/blob/a3f393bc95784320fd21d0871116d33dcb72982a/tensorflow/python/data/experimental/kernel_tests/serialization/BUILD#L99).

Reply:
Ah, it's not about bazel. Yeah, it works to use duplicate tags in all tests, but just too many duplicate codes/lines and bug numbers. 

As almost all of our tests using the same two tags, it will simplify the build file if a `common_tag` is created and used in each test. Will it be better to have something like:

```
COMMON_TAGS = [
   "no_pip",  # b/161253163
   "no_windows",  # b/160628318
]

py_test(
    name = "antirectifier_benchmark_test",
    srcs = ["keras_examples_benchmarks/antirectifier_benchmark_test.py"],
    python_version = "PY3",
    tags = COMMON_TAGS,
    deps = ....
)

py_test(
    name = "another_benchmark_test",
    srcs = ..., 
    python_version = "PY3",
    tags = COMMON_TAGS,
    deps = ....
)
```

Reply:
Thanks a lot! It's very helpful!